qemu/tests/remove_pending_watches_after_unplug_port.py

```diff
@@ -0,0 +1,69 @@
+import os
+import time
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import utils_test
+from qemu.tests.virtio_serial_file_transfer import generate_data_file, copy_scripts
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Remove pending watches after virtserialport unplug.
+
+     1) Start guest with virtio serial device(s).
+     2) Open the chardev on the host
+     3) Send 2g file from guest to host
+     4) Hot-unplug the port on the host
+     5) After step 4, read transferred data on host
+     6) Guest has no crash or panic
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    file_size = int(params.get("file_size"))
+    os_type = params["os_type"]
+    session = vm.wait_for_login()
+    char_path = params.get("chardev_path")
+    guest_dir = params.get("tmp_dir", '/var/tmp/')
+    port_name = params["file_transfer_serial_port"]
+    guest_path = params.get("guest_script_folder", "C:\\")
+    guest_scripts = params.get("guest_scripts",
+                               "VirtIoChannel_guest_send_receive.py")
+    host_listen = open(char_path, 'w+')
+    copy_scripts(guest_scripts, guest_path, vm)
+    guest_script = os.path.join(guest_path, params['guest_script'])
+    python_bin = '`command -v python python3 | head -1`'
+    if os_type == "windows":
+        driver_name = params["driver_name"]
+        session = vm.wait_for_login()
+        session = utils_test.qemu.windrv_check_running_verifier(
+            session, vm, test, driver_name)
+    guest_file_name = generate_data_file(guest_dir, file_size, session)
+    guest_cmd = ("%s %s -d %s -f %s -a %s" %
+                 (python_bin, guest_script,
+                  port_name, guest_file_name, 'send'))
+    try:
+        guest_md5 = session.cmd_status_output(guest_cmd)[1]
```

Comment:
Literally, `guest_md5`  means the md5 value of some thing, but `cmd_status_output[1]` not mean that.
the output[1] is something like "md5_sum = ******"  if no error.

Reply:
I change the step of the case according to related bug so we don't need this, thanks.